### DISCUSSION TITLE
Any ways to use Alpine offline without CDN and without import module?

### DISCUSSION THREAD

**Discussion by sawirricardo** [alpinejs/alpine#2338] [1. Help]
From the docs, I only see 2 ways
1. Include the cdn
2. Import module

But, due to legacy projects, I need to download this repo (so users can use offline). And I don't see how we can use it offline, but not using that module way. I've tried to import directly this repo, but errors are occuring everywhere.

Thank you

**Comment by SimoTod** [comment]
You can copy the cdn link in your browser, click save and copy it into your project. Your users need a modern browser though, alpine doesn't support Explorer (just in case, since you wrote legacy project)
